app/models/database_backup.rb

```diff
@@ -5,9 +5,9 @@ class DatabaseBackup < ApplicationRecord
   }.freeze
 
   def self.backup_supported?
-    # We currently only support Postgres via internal/external db
     return @backup_supported unless @backup_supported.nil?
-    @backup_supported = MiqDbConfig.current.options[:name] =~ /[in|ex]ternal|postgresql/ ? true : false
+    db_config = Rails.configuration.database_configuration[Rails.env]
+    @backup_supported = db_config["adapter"] == "postgresql"
```

Comment:
I was thikning the same thing...I might just kill the backup_supported method altogether.